app/controllers/users_controller.rb

```diff
@@ -479,6 +479,8 @@ def is_local_username
 
     raise Discourse::InvalidParameters.new(:usernames) if !usernames.kind_of?(Array)
 
+    usernames = usernames[0..20]
```

Comment:
What do you think about raising an error with the right error message to the client instead of silently truncating here? I think this will help to prevent surprises in the future.